service/src/main/kotlin/fi/espoo/evaka/incomestatement/IncomeStatementControllerCitizen.kt

```diff
@@ -211,19 +211,27 @@ class IncomeStatementControllerCitizen(private val accessControl: AccessControl)
         user: AuthenticatedUser.Citizen,
         clock: EvakaClock,
         @RequestBody body: IncomeStatementBody,
+        @RequestParam draft: Boolean?,
```

Comment:
Miksi draft muuttujalla on kolme tilaa (`true`, `false` ja `undefined`)? Eikö `true` ja `false` riittäisi?

Reply:
Varmaan taaksepäin yhteensopivuus deployn yhteydessä.

Reply:
Kyllähän noita formeja näytetään tallennettavan muutama kymmenen vuorokaudessa, joten kyllä tämä väliaikainen undefined:in tukeminen voi muutaman yhmisen päivää helpottaa.

Reply:
Tämmöisiin voisi sitten laittaa TODO kommentin, että tämä on sen takia nullable. Muuten kun nämä unohtuu siivota, niin ei enää selviä kuin laajemmalla koodin katselmoinnilla, että tarviiko olla nullable vai ei.

Reply:
teen saman tien jatkopullan